include/aspect/postprocess/geoid.h

```diff
@@ -119,6 +119,11 @@ namespace aspect
         double density_above;
         double density_below;
 
+        /**
+         * A parameter to control whether to turn off the dynamic topography contribution on geoid
+         */
+        bool remove_dynamic_topo_contribution;
```

Comment:
May I suggest to invert the logic of this parameter? I would prefer to call the parameter `include_dynamic_topo_contribution` with a default value of `true`. The reasoning for this is that below you have a lot of conditions that say `if remove_dynamic_topo_contribution == false`, this suggests you really want to test for `if include_dynamic_topo_contribution == true`. Also my brain hurts every time I try to understand double negations :smile:. You would need to update the descriptions and the if statements below, but I think it will be easier to understand afterwards.